docs/text_summaries.ipynb

```diff
@@ -294,18 +355,54 @@
         "\"\"\"\n",
         "\n",
         "with file_writer.as_default():\n",
-        "  tf.summary.text(\"Run Params\", pretty_json(some_obj_worth_noting), step=0)\n",
-        "  tf.summary.text(\"Markdown jubiliee\", markdown_text, step=0)\n",
+        "  tf.summary.text(\"run_params\", pretty_json(some_obj_worth_noting), step=0)\n",
+        "  tf.summary.text(\"markdown_jubiliee\", markdown_text, step=0)\n",
         "    "
       ]
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 10,
       "metadata": {
         "id": "57082d8d6839"
       },
-      "outputs": [],
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "Reusing TensorBoard on port 6008 (pid 44187), started 1 day, 16:31:45 ago. (Use '!kill 44187' to kill it.)"
```

Comment:
I'd probably omit this cached output as well?

Reply:
Dropped all of them.